### PR TITLE
Add teach google pictionary option

### PR DESCRIPTION
**Pull request by mattmcan**
@casey-collab please review my changes.

### CODE REVIEW

File: english/pictionary/pictionary.md [hunk 1]
```diff
@@ -0,0 +1 @@
+You decide to teach google to be a [Pictionary](https://quickdraw.withgoogle.com/) champion.
```
Comment (casey-collab) [inline english/pictionary/pictionary.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by mattmcan** [comment]
@casey-collab Please review my changes.

**Comment by casey-collab** [comment]
Looks great! :seedling:

**Pull request merged**